### ISSUE TITLE
Issue in installing pymeeus

### ISSUE THREAD

**Issue by dheerajkumar-solanki** [architest/pymeeus#8]
While installing pymeeus, it is giving following error:

```
pip install pymeeus

Collecting pymeeus
  Using cached https://files.pythonhosted.org/packages/e4/45/04ce4324d00e27a491d52ae40e1b6ba035f9f21e16e478d4f3d3f19ddbca/PyMeeus-0.3.11.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-wFGXTS/pymeeus/setup.py", line 12, in <module>
        with open(path.join(here, 'README.rst'), encoding="utf8") as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-wFGXTS/pymeeus/
```

If I install lower version like 0.3.10, it is installing.

**Comment by bwangelme** [comment]
I also meet this question.

**Comment by architest** [comment]
The "encoding" keyword was recently introduced by a pull request that tried to solve a problem with the encoding of README.rst. However, it seems that it produces undesired side-effects and I decided to remove it and deal with the README problem directly.

I've released a new PyMeeus version (0.3.12) that should not have this problem.
